docs/services/blockstorage.rst

```diff
@@ -1,89 +1,202 @@
 .. _block_storage:
 
-block-storage
+=============
+Block storage
 =============
 
-Commands for Rackspace Cloud Block Storage.
+This section lists all block storage commands supported by Rackspace CLI
 
-Synopsis
+Commands
 --------
 
-::
+All block storage commands are based on this syntax::
 
    rack block-storage <resource> <action> [command flags]
 
-Commands
---------
+*Command flags* allow you to customize certain attributes of the command.
+Below is a table of some of the most common *command flags*:
+
++-----------------+--------------------------------------+
+| `Flags`         | Description                          |
++-----------------+--------------------------------------+
+| `--size`        | Allows you to enter the size you     |
+|                 | want the volume to be. Valid values  |
+|                 | are between 75 and 1024.             |
++-----------------+--------------------------------------+
+| `--name`        | Allows you to name the volume or     |
+|                 | snapshot.                            |
++-----------------+--------------------------------------+
+| `--description` | Allows you to leave a description    |
+|                 | of the volume or snapshot.           |
++-----------------+--------------------------------------+
+| `--volume-type` | Allows you to enter a volume type.   |
++-----------------+--------------------------------------+
+| `--rename`      | Allows you to rename a volume.       |
++-----------------+--------------------------------------+
```

Comment:
Let's not add tables anywhere. Please use bulleted lists when there's just a two-column table.